### PR TITLE
Sf2 slurm

### PR DESCRIPTION
**Pull request by andrewzhang505**
Added instructions for running slurm to docs
Fixed installation for mujoco

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -45,7 +45,7 @@
     extras_require={
         "dev": ["black", "isort", "pytest<8.0", "flake8", "pre-commit"],
         "atari": ["gym[atari, accept-rom-license]"],
-        "mujoco": ["gym[mujoco, mujoco_py]"],
+        "mujoco": ["gym[mujoco]", "mujoco", "mujoco_py>=2.0,<3.0"],
```
Comment (wmFrank) [inline setup.py:48]: Why do we need to set the version? I think gym will automatically import the latest mujoco.

**Review by wmFrank — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (andrewzhang505) [reply setup.py:48]: The 2nd mujoco and mujoco_py are not from gym and need to be installed seperately. The versioning on mujoco_py is probably not necessary but I included it anyways

Comment (alex-petrenko) [reply setup.py:48]: I wonder why tests worked without it :)

Btw I think restricting the major version of a package is a good practice @wmFrank 
The reason is that major version updates usually introduce incompatible API changes. Unless we want our software to break every time authors of another package release a new major version, we should use limits like this.

One example is going to be sample factory 2.0. If someone has just sample_factory in their dependencies, us releasing a new PyPI package is going to break their library.

File: docs/get-started/running-slurm.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+Activate your conda environment with `bash` and `conda activate sf2` then `cd sample-factory`
+
+Run your runner script - an example mujuco runner (replace run and slurm_workdir with appropriate values)
+```
+python -m sample_factory.runner.run --runner=slurm --slurm_workdir=./slurm_mujoco --experiment_suffix=slurm --slurm_gpus_per_job=1 --slurm_cpus_per_gpu=16 --slurm_sbatch_template=./sample_factory/runner/slurm/sbatch_template.sh --pause_between=1 --slurm_print_only=False --run=sf_examples.mujoco_examples.experiments.mujoco_all_envs
+```
+
+### Other Helpful Commands
+
+`squeue` - List all jobs running or queued in the cluster
```
Comment (alex-petrenko) [inline docs/get-started/running-slurm.md:44]: @andrewzhang505 sorry that I wasn't completely clear about this.
I think we should have an instruction for running SF2 on Slurm clusters, not specifically on brain :)
Can you please update this instruction to not mention brain or USC, but be generally for Slurm?

We can keep this instruction for internal use, or you can share it with our labmates on compute-forum!

I will merge your PR anyway for the Mujoco change but please update it in another PR!
Also, while we're at it, can you please add documentation website link to readme? I want to see how this looks on the website :)

**Review by alex-petrenko — APPROVED** [review]

**Pull request merged**